azure-pipelines.yml

```diff
@@ -138,7 +138,7 @@ jobs:
         # clone azure-cli
         git clone --single-branch -b dev https://github.com/Azure/azure-cli.git ../azure-cli
 
-        pip install azdev
```

Comment:
These lines are duplicated code and should be unified like https://github.com/Azure/azure-cli/blob/f46d3bed2dec22e7ea12e5543dec9363d94350ae/.azure-pipelines/templates/azdev_setup.yml